roles/nova-data/tasks/libvirt.yml

```diff
@@ -12,7 +22,7 @@
     - libvirt-bin={{ nova.libvirt_bin_version }}
     - python-libvirt={{ nova.python_libvirt_version }}
     - qemu-kvm={{ nova.qemu_kvm_version }}
-    - qemu-system-x86={{ nova.qemu_kvm_version }}
+    - qemu-system-{{ fact_arch }}={{ nova.qemu_kvm_version }}
```

Comment:
I'd rather see this as a variable like `nova.qemu_system_package: qemu-system-x86` in defaults.yml and overridden in the ansible inventory for power nodes.